spec/support/approved_proposals_helpers.rb

```diff
@@ -78,5 +78,20 @@ def mock_urs_get_users(count: 1, users: [])
       urs_response = cmr_success_response(urs_response_body.to_json)
       allow_any_instance_of(Cmr::UrsClient).to receive(:get_urs_users).and_return(urs_response)
     end
+
+    # Fake getting ACLs from CMR, then getting groups from CMR, then getting group members from CMR
+    def mock_get_approver_emails
```

Comment:
Mocking most of this is fine for some of the tests, but we should have at least one instance where we are running through as many of the endpoints we can talk to in test (i.e. CMR) as possible.